lib/plugins.ts

```diff
@@ -1,7 +1,9 @@
-var webdriver = require('selenium-webdriver');
-import {Logger} from './logger';
-import * as Q from 'q';
+import * as q from 'q';
```

Comment:
Thanks for fixing this. This is leftover from experimenting to get the 'Q' module to work with our typings.